src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerPlugin.java

```diff
@@ -188,7 +188,7 @@ public PerformanceAnalyzerPlugin(final Settings settings, final java.nio.file.Pa
 
         scheduledMetricCollectorsExecutor.addScheduledMetricCollector(new MetricsPurgeActivity());
 
-        scheduledMetricCollectorsExecutor.addScheduledMetricCollector(new NodeDetailsCollector());
```

Comment:
Can we remove this default constructor from `NodeDetailsCollector`, or is it required?

Reply:
Let me check the usages. Maybe the no arg constructor is not used anymore.